src/libcrun/custom-handler.c

```diff
@@ -248,6 +248,16 @@ libcrun_configure_handler (struct custom_handler_manager_s *manager,
   *out = NULL;
   *cookie = NULL;
 
+  // Kubernetes sandbox containers must be executed as regular process
+  // Example sandbox container can contain pause process
+  // See: https://github.com/containers/crun/issues/798
+  // before invoking handler check if this is not a kubernetes sandbox
+  annotation = find_annotation (container, "io.kubernetes.cri.container-type");
+  if (strcmp (annotation, "sandbox") == 0)
+    {
+      return 0;
+    }
```

Comment:
since it is a single statement, we do not need `{` `}`

Reply:
fixed it.